interop/shared/src/main/scala/scalaz/zio/interop/future.scala

```diff
@@ -16,6 +17,22 @@ object future {
       }
   }
 
+  implicit class FiberObjOps(private val fiberObj: Fiber.type) extends AnyVal {
+    def fromFuture[A](_ftr: => Future[A])(ec: ExecutionContext): Fiber[Throwable, A] =
```

Comment:
Strictly speaking, the laziness is not required, but nonetheless, I can see it would be useful in some cases. Do you want to make `ec` implicit or not?

Reply:
I think this should be the same as `IO.fromFuture`, which doesn't make `ec` implicit.